README.md

```diff
@@ -1,6 +1,8 @@
 Torrentor!
 =============
 
+![Torrentor app](torrentor.png "Torrentor")
```

Comment:
While you're updating the readme, a few things to improve:

"To build Torrentor you need some libs which are optional packages."
-> "To build Torrentor you need some development libraries:"

* OpenSSL   -- ```$ pkgman install devel:libssl```
* libevent  -- ```$ pkgman install devel:libevent```
* curl      -- ```$ pkgman install devel:libcurl```

Also add a linebreak in the next to last line a set full-stops.

Reply:
Updated

Reply:
You forgot to change the "lib:" parts of the pkgman lines to "devel:". To build the application we need the dev libs, not the runtime libs.

Reply:
Ah, updated.